packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -0,0 +1,212 @@
+import { ThemingParameters } from '@ui5/webcomponents-react-base/lib/ThemingParameters';
+import { useConsolidatedRef } from '@ui5/webcomponents-react-base/lib/useConsolidatedRef';
+import { enrichEventWithDetails } from '@ui5/webcomponents-react-base/lib/Utils';
+import { ChartContainer } from '@ui5/webcomponents-react-charts/lib/components/ChartContainer';
+import { XAxisTicks } from '@ui5/webcomponents-react-charts/lib/components/XAxisTicks';
+import { YAxisTicks } from '@ui5/webcomponents-react-charts/lib/components/YAxisTicks';
+import { LineChartPlaceholder } from '@ui5/webcomponents-react-charts/lib/LineChartPlaceholder';
+import { useLegendItemClick } from '@ui5/webcomponents-react-charts/lib/useLegendItemClick';
+import React, { CSSProperties, FC, forwardRef, Ref, useCallback, useMemo } from 'react';
+import {
+  CartesianGrid,
+  Legend,
+  Scatter,
+  ScatterChart as ScatterChartLib,
+  ReferenceLine,
+  Tooltip,
+  XAxis,
+  YAxis,
+  ZAxis
+} from 'recharts';
+import { useChartMargin } from '../../hooks/useChartMargin';
+import { useLongestYAxisLabel } from '../../hooks/useLongestYAxisLabel';
+import { usePrepareDimensionsAndMeasures } from '../../hooks/usePrepareDimensionsAndMeasures';
+import { useTooltipFormatter } from '../../hooks/useTooltipFormatter';
+import { IChartBaseProps } from '../../interfaces/IChartBaseProps';
+import { IChartMeasure } from '../../interfaces/IChartMeasure';
+import { defaultFormatter } from '../../internal/defaults';
+import { tickLineConfig, tooltipContentStyle, tooltipFillOpacity, xAxisPadding } from '../../internal/staticProps';
+import { useObserveXAxisHeights } from '../../hooks/useObserveXAxisHeights';
+
+interface MeasureConfig extends IChartMeasure {
+  opacity?: number;
+  /**
+   * Defines axis of measures
+   */
+  axis: 'x' | 'y' | 'z';
+}
+
+interface ScatterDataObject {
+  label?: string;
+  data?: any[];
+  color?: CSSProperties['color'];
+}
+
+export interface ScatterChartProps extends IChartBaseProps {
+  dataset?: ScatterDataObject[];
+  /**
+   * An array of config objects. Each object is defining one line in the chart.
+   *
+   * <h4>Required properties</h4>
+   * - `accessor`: string containing the path to the dataset key this line should display. Supports object structures by using <code>'parent.child'</code>.
+   *   Can also be a getter.
+   *
```

Comment:
axis should be documented as required attribute.
formatter should be documented as optional attribute
label should be documented as well